client/my-sites/plans/current-plan/index.jsx

```diff
@@ -101,23 +101,24 @@ class CurrentPlan extends Component {
 		} = this.props;
 
 		const currentPlanSlug = selectedSite.plan.product_slug,
-			isLoading = this.isLoading();
+			isLoading = this.isLoading(),
+			isJetpack = selectedSite && selectedSite.jetpack;
```

Comment:
Completely optional, but we can also pass through isJetpack as a prop in connect:
```javascript
export default connect(
	( state, ownProps ) => {
		const selectedSite = getSelectedSite( state );
		const selectedSiteId = getSelectedSiteId( state );
                const isJetpack = selectedSite && selectedSite.jetpack;
                //or via another selector:
                //const isJetpack = isJetpackSite( site, selectedSiteId )
		return {
			selectedSite,
			selectedSiteId,
                        isJetpack,
```